ch5/tutorial/crud_query.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import FastAPI, HTTPException
+
+# Create a FastAPI instance
+app = FastAPI()
+
+# User database
+USER_DB = {}
+
+# Fail response
+NAME_NOT_FOUND = HTTPException(status_code=400, detail="Name not found.")
+
+
+@app.post("/users")
+def create_user(name: str, nickname: str):
+    USER_DB[name] = nickname
+    return {"status": "success"}
+
+
+@app.get("/users")
+def read_user(name: str):
+    if name not in USER_DB:
+        raise NAME_NOT_FOUND
+    return {"nickname": USER_DB[name]}
+
+
+@app.put("/users")
+def update_user(name: str, nickname: str):
+    if name not in USER_DB:
+        raise NAME_NOT_FOUND
+    USER_DB[name] = nickname
+    return {"status": "success"}
+
+
+@app.delete("/users")
+def delete_user(name: str):
+    if name not in USER_DB:
+        raise NAME_NOT_FOUND
+    USER_DB.pop(name)
```

Comment:
```suggestion
    del USER_DB[name]
```

del을 사용하는 쪽으로 수정하면 좋을 것 같습니다!

Reply:
반영하였습니다!